diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc9a268e..643f3ab3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,6 +105,8 @@ if(SFML_OS_IOS)
     sfml_set_option(SFML_IOS_DEPLOYMENT_TARGET "10.2" STRING "The minimal iOS version that will be able to run the built binaries. Cannot be lower than 10.2.")

     sfml_set_option(SFML_CODE_SIGN_IDENTITY "iPhone Developer" STRING "The code signing identity to use when building for a real device")
+elseif(PROCURSUS)
+    sfml_set_option(SFML_IOS_DEPLOYMENT_TARGET "14.0" STRING "The minimal iOS version that will be able to run the built binaries. Cannot be lower than 10.2.")
 endif()

 # Android options
@@ -152,7 +154,7 @@ if(SFML_OS_WINDOWS OR SFML_OS_IOS)
     set(DEFAULT_INSTALL_MISC_DIR .)
 elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OS_OPENBSD OR SFML_OS_NETBSD)
     set(DEFAULT_INSTALL_MISC_DIR share/SFML)
-elseif(SFML_OS_MACOSX)
+elseif(SFML_OS_MACOSX OR PROCURSUS)
     set(DEFAULT_INSTALL_MISC_DIR /usr/local/share/SFML)
 elseif(SFML_OS_ANDROID)
     set(DEFAULT_INSTALL_MISC_DIR ${CMAKE_ANDROID_NDK}/sources/third_party/sfml)
